infra/cifuzz/cifuzz.py

```diff
@@ -201,6 +211,55 @@ def run_fuzzers(fuzz_seconds, workspace):
   return True, False
 
 
+def get_lastest_build_version(project_name):
+  """Gets the latest OSS-Fuzz build version for a projects fuzzers.
+
+  Args:
+    project_name: The name of the project who's build is being retrieve.
+
+  Returns:
+    A string with the latest build version or None.
+  """
+  http_get_string = os.path.join(GCS_BASE_URL, project_name,
+                                 project_name + '-address-latest.version')
```

Comment:
Let's do this instead.

```
sanitizer = 'address'
version = {project_name}-{sanitizer}-latest.version'.format(project_name=project_name, sanitizer=sanitizer)`
version_url = posixpath.join(GCS_BASE_URL, project_name, version)
```
`version_url` is much more descriptive than `http_get_string`
I hackily `posixpath.join` for this sort of URL manipulation. If you can find anything better please use it (probably isn't bad to do string formatting frankly).
But posixpath is better than os.path because URLs are meant to look like posixpaths which use `/`  for hierarchy. On windows os.path.join uses `\` which would be wrong.